components/grid/sizing.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Grid - Sizing
+page_title: Grid Size | Grid for Blazor
+description: Adjust the size and modify the appearance of the Telerik UI for Blazor Grid.
+slug: grid-sizing
+tags: telerik,blazor,grid,size,compact, dense, small
+published: true
+position: 47
+---
+
+# Grid Size
+
+This feature attempts to address the need for a **Compact Grid**, which renders more items by utilizing the available space, mainly through setting smaller padding in its cells.
+
+Changing the [`Size`](#size) property affects different building blocks of the component (tables, buttons, inputs, dropdowns, and others). 
+
+## Size
+
+The `Size` property accepts values of type [`ThemeConstants.Grid.Size`](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.ThemeConstants.Grid.Size) and the supported values are: 
+
+| Class members | Manual declarations |
+|------------|--------|
+|`Small`|`sm`|
+|`Medium` <br /> default value|`md`|
+
+
+>caption The built-in Size modes
+
+```CSHTML
+@* These are all built-in Size modes *@
+
+@{ 
+    var fields = typeof(Telerik.Blazor.ThemeConstants.Grid.Size)
+        .GetFields(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static
+        | System.Reflection.BindingFlags.FlattenHierarchy)
+        .Where(field => field.IsLiteral && !field.IsInitOnly).ToList();
+
+    foreach (var field in fields)
+    {
+        string size = field.GetValue(null).ToString();
+
+        <TelerikGrid Size="@size"
+					 Data="@MyData"
+			 	     Height="350px">
+            <GridColumns>
+                <GridColumn Field="ID"></GridColumn>
+                <GridColumn Field="TheName" Title="Employee Name"></GridColumn>
+            </GridColumns>
+        </TelerikGrid>
+        <br />
+    }
+}
+@code {
+    public IEnumerable<object> MyData = Enumerable.Range(1, 50).Select(x => new { ID = x, TheName = "name " + x });
+}
+```
+
+## Notes
+
+1. The `Size` option does not affect elements displayed inside a PopUp (such as [Filter Menu](https://docs.telerik.com/blazor-ui/components/grid/filter/filter-menu), [Column Menu](https://docs.telerik.com/blazor-ui/components/grid/columns/menu), etc.). It is this way since the PopUps are rendered on root level and the `Size` option cannot propagate to them. Тo change the `Size` options of the elements inside PopUps as well, you can use a template and explicitly specify their `Size`.
```

Comment:
```suggestion
1. The `Size` option does not affect elements displayed inside a PopUp (such as [Filter Menu]({%slug grid-filter-menu%}), [Column Menu]({%slug grid-column-menu%}), etc.). By design, all PopUp elements are rendered on root level, so they are not technically inside the Grid. Thus, the `Size` option cannot propagate to them. Тo change the `Size` options of the elements inside PopUps, you can use a template(where available), so you can override the built-in rendering. You can then add custom elements and explicitly specify their `Size`.
```
Wording upgrade. Use slugs instead of links.